docs/_packages/events_api.md

```diff
@@ -366,7 +366,7 @@ slackEvents.on('reaction_added', (event, respond) => {
   console.log('Reaction event received');
   // Normal success
   respond();
-})
+});
```

Comment:
Is this correct? I'm only seeing an addition of this semicolon as the only change.

Reply:
Yes, it is. Without the semicolon, the code doesn't work 😢

Reply:
weird. I have approved it